src/Payment/Service.php

```diff
@@ -37,4 +45,11 @@ public function customers(): CustomerService;
      * @return RefundService
      */
     public function refunds(): RefundService;
+
+    /**
+     * Return the name of the provider.
+     *
+     * @return string
+     */
+    public function getProviderName(): string;
```

Comment:
we shouldn't need this?

Reply:
On the api3 we are recording the transaction provider (`transaction_provider`  field of billing_events table) it was just a way to prevent us having 'stripe' statement in the PaymentService on the API3 I've created. And we won't have to change the provider name in the PaymentService if we'd change it.